test/Microsoft.AspNet.Mvc.Core.Test/ApplicationModel/ActionModelTest.cs

```diff
@@ -78,6 +79,13 @@ public void CopyConstructor_CopiesAllProperties()
                     // Ensure non-default value
                     Assert.NotEmpty((IEnumerable<object>)value1);
                 }
+                else if (typeof(IDictionary<object, object>).IsAssignableFrom(property.PropertyType))
```

Comment:
Couldnt' we simply check the actual properties rather than doing reflection? Seems like there's about half a dozen of these so shouldn't be hard to just test them out.